### DISCUSSION TITLE
Welcome to TheXTech Discussions!

### DISCUSSION THREAD

**Discussion by Wohlstand** [Wohlstand/TheXTech#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GooViR** [comment]
i remember like over a year ago now i made this really terrible buttons of the android version of this i ont have an android phone anymore so i dont know if there still being used but yeah... story time

**Reply by GooViR** [reply]
oh my god i just went digging in the files and they are lol

**Comment by cinnamon189** [comment]
I used to be a regular of the SMBX 2 community, but did not play any SMBX or SMBX 2 episodes for years. A couple of months ago I switched from Windows to Linux, and to my dismay, it's difficult to get SMBX 2 to run on Linux Mint. Then I remembered TheXTech and am now hoping to play some of the episodes again that I really like. Fingers crossed that TheXTech will eventually become SMBX 2-compatible... My first contact with this community was when I erroneously posted a Moondust bug in the TheXTech bug tracker :-P My contributions are going to be of linguistic nature.

Also, @Wohlstand : Maybe make this thread a sticky, so that new community members find it more easily? That is, I'm assuming that GitHub discussions feature sticky threads.

**Comment by Vinicin2** [comment]
Well, I also hope that ThexThec improves a lot. I stopped playing SMBX 2 a long time ago because it was full of bugs, and ThexThec also has many improvements.I see that ThexThec is evolving every day, which is good because it's a direct continuation of smbx 1.3, without any mods, but rather a perfect continuation and evolution, always offering new things.Still supporting Lua and online mode, and implementing many other features.
